include/applicationLayer.h

```diff
@@ -0,0 +1,12 @@
+#ifndef APPLICATION_LAYER_H_INCLUDED
+#define APPLICATION_LAYER_H_INCLUDED
+/*
+       Wenn Result.error_code = SUCCESS steht im Result.value der Wert den der
+       Client auf der Konsole sehen soll.
+       Ansonsten wird nur der error_code im Result �bermittelt und das value
+       bleibt LEER.
+*/
+//Entscheidet ob und welcher Befehl ausgef�hrt wird, gibt Result an ClientSession zur�ck. M�gliche Error: COMMAND_NOT_FOUND
+Result executeCommand(Command command);

```

Comment:
Sry that I didn't pay attention to this before. But the applicationLayer header depends on the utils header, because it uses both structs `Result` and `Command`. So for the sake of completeness this header should include the utils header.